src/debuggability.js

```diff
@@ -117,15 +117,15 @@ Promise.enableHadronTestTracking = function (enable) {
 
 var disableLongStackTraces = function() {};
 Promise.longStackTraces = function () {
-    if (async.haveItemsQueued() && !config.longStackTraces) {
+    if (async.areItemsQueued() && !config.longStackTraces) {
```

Comment:
I might be reading it wrong, but should be this saying "if we have items queued and config says to use longstacktraces, throw error"? The "!config.longStackTraces" would mean that longstacktraces are not set in config right?